Classes/Queue/SerialQueue.php

```diff
@@ -0,0 +1,129 @@
+<?php
+namespace Flowpack\JobQueue\Common\Queue;
+
+/*
+ * This file is part of the Flowpack.JobQueue.Common package.
+ *
+ * (c) Contributors to the package
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Core\Booting\Scripts;
+use TYPO3\Flow\Utility\Algorithms;
+
+class SerialQueue implements QueueInterface
+{
+    /**
+     * @Flow\InjectConfiguration(package="TYPO3.Flow")
+     * @var array
+     */
+    protected $flowSettings;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @param string $name
+     * @param array $options
+     */
+    public function __construct($name, array $options = [])
+    {
+        $this->name = $name;
+    }
+
+    /**
+     * @inheritdoc
```

Comment:
I personally prefer not using `@inheritdoc` because I usually only read documentation in my IDE, not on an online / rendered page. But maybe that's just me.